drivers/copy/copy_linux.go

```diff
@@ -203,6 +203,9 @@ func DirCopy(srcDir, dstDir string, copyMode Mode, copyXattrs bool) error {
 			if err != nil {
 				return err
 			}
+			if unixListener, ok := s.(*net.UnixListener); ok {
```

Comment:
I think this should fail if `!ok`; or, maybe better, use `net.ListenUnix` explicitly to make the cast unnecessary.